options/locale/locale_en-US.ini

```diff
@@ -112,6 +112,7 @@ step2 = Step 2:
 
 error = Error
 error404 = The page you are trying to reach either <strong>does not exist</strong> or <strong>you are not authorized</strong> to view it.
+back = Back
```

Comment:
"back" has different meaning .... I think the key could be "goto_back"

Reply:
Renamed to go_back